ksqldb-functional-tests/src/test/resources/query-validation-tests/topk-group-by.json

```diff
@@ -95,6 +95,118 @@
         {"topic": "S2", "key": 0, "value": {"TOPK":["c","b","b"]}},
         {"topic": "S2", "key": 0, "value": {"TOPK":["d","c","b"]}}
       ]
+    },
+    {
+      "name": "topk integer struct",
+      "format": ["AVRO", "JSON", "PROTOBUF", "PROTOBUF_NOSR"],
+      "statements": [
+        "CREATE STREAM TEST (ID BIGINT KEY, NAME varchar, VALUE integer) WITH (kafka_topic='test_topic',value_format='{FORMAT}');",
+        "CREATE TABLE S2 as SELECT ID, topk(value, name, 3) as topk FROM test group by id;"
+      ],
+      "inputs": [
+        {"topic": "test_topic", "key": 0, "value": {"NAME": "zero", "key": 0, "value": 0}},
+        {"topic": "test_topic", "key": 0, "value": {"NAME": "one", "key": 0, "value": 100}},
+        {"topic": "test_topic", "key": 0, "value": {"NAME": "two", "key": 0, "value": 99}},
+        {"topic": "test_topic", "key": 0, "value": {"NAME": "three", "key": 0, "value": 7}},
+        {"topic": "test_topic", "key": 0, "value": {"NAME": "four", "key": 0, "value": 100}}
```

Comment:
Add an entry where the name is null

Reply:
I added tests where the name is null, or the value is null.